lib/widgets/dialog.dart

```diff
@@ -112,3 +116,75 @@ DialogStatus<bool> showSuggestedActionDialog({
       ]));
   return DialogStatus(future);
 }
+
+bool debugDisableBetaCompleteDialog = false;
+void resetDebugDisableBetaCompleteDialog() {
```

Comment:
nit: this should also start with "debug", as it's meant to be called only in debug mode